### ISSUE TITLE
/wrf/gcc/WRFV3/chem folder 

### ISSUE THREAD

**Issue by mabelsfatalfable** [yyr/wrf_install#25]
The folder "chem" in "/wrf/gcc/WRFV3/" is needed to be created in before compiling "./build_app.sh WRF" for the default type installation.

**Comment by yyr** [comment]
Is it to install WRF chem?
I never built chem with these scripts

**Comment by mabelsfatalfable** [comment]
No, the installation part does not include the WRF-Chem option, but it tries to read/write on /wrf/gcc/WRFV3/chem directory at some point doing loops during the building of WRF

I will check the logs again, the other parts of the installation works by default without any problem.

**Comment by yyr** [comment]
Now, I do remember make looking for chem. I ignored them since they are not show stoppers. That must be bug in WRF Makefiles.

We can silence them if it is needed by creating  a empty folder in wrf.sh

**Comment by mabelsfatalfable** [comment]
Yes, it is what I did during the installation, It will sure solve the problem encountered.
